security.md

```diff
@@ -4,35 +4,39 @@ title: Reporting a Security Vulnerability
 permalink: /security/
 ---
 
-# Reporting a Security Vulnerability
+# Security Policy
 
-The WSO2 security team welcomes contributions from our user community, developers, and security researchers to reinforce our product security. The security team will be more than happy to assist you in such efforts.
+Ballerina project maintainers take security issues very seriously and all the vulnerability reports are treated with the highest priority and confidentiality.
 
-We strongly encourage you to report security vulnerabilities to our private security mailing list: [security@ballerina.io](mailto:security@ballerina.io) - first, before disclosing them in any public forums.
+- [Reporting a vulnerability](#reporting-a-vulnerability)
+- [Handling a vulnerability](#handling-a-vulnerability)
 
-This is a private mailing list where only members of the WSO2 internal security team are subscribed to, and is treated as top priority.
+## Reporting a vulnerability
 
-If you wish to send secure messages to [security@ballerina.io](mailto:security@ballerina.io), you may use the following key:
+Ensure you are using the latest Ballerina version before you run an automated security scan or perform a penetration test against them.
 
-security@ballerina.io: 0168 DA26 2989 0DB9 4ACD  8367 E683 061E 2F85 C381 [pgp.mit.edu](https://pgp.surfnet.nl/pks/lookup?op=vindex&fingerprint=on&search=0xE683061E2F85C381)
+If you have any concerns regarding the security aspects of the source code or any other resource in this repo or have uncovered a security vulnerability, we strongly encourage you to report that to our private and highly confidential security mailing list: **[security@ballerina.io](mailto:security@ballerina.io)** first using the below key without disclosing them in any forums, sites, or other groups - public or private. 
```

Comment:
Shall we break this into multiple sentences as in [1] for more clarity

[1] https://docs.wso2.com/display/Security/WSO2+Security+Vulnerability+Reporting+Guidelines

Reply:
Changed as follows:

Based on the ethics of responsible disclosure, you must only use the **security@ballerina.io** mailing list to report security vulnerabilities and any other concerns regarding the security aspects of the source code or any other resource in this repo.

**Warning:** To protect the end-user security, please do not use any other medium to report security vulnerabilities. Also, kindly refrain from disclosing the vulnerability details you come across with other individuals, in any forums, sites, or other groups - public or private before it’s mitigation actions and disclosure process are completed.

Use the following key to send secure messages to security@ballerina.io:

security@ballerina.io: 0168 DA26 2989 0DB9 4ACD 8367 E683 061E 2F85 C381 [pgp.mit.edu](https://pgp.surfnet.nl/pks/lookup?op=vindex&fingerprint=on&search=0xE683061E2F85C381)